deepchecks/tabular/checks/train_test_validation/train_test_feature_drift.py

```diff
@@ -83,6 +83,11 @@ class TrainTestFeatureDrift(TrainTestCheck, ReduceFeatureMixin):
     categorical_drift_method: str, default: "cramer_v"
         decides which method to use on categorical variables. Possible values are:
         "cramer_v" for Cramer's V, "PSI" for Population Stability Index (PSI).
+    relative_change: bool, default: False
```

Comment:
Are you sure we want to add this to feature drift? Do you think that in *features* we may have a situation where all categories should weigh the same?